subworkflows/ebi-metagenomics/mapseq_otu_krona/main.nf

```diff
@@ -0,0 +1,46 @@
+
+include { MAPSEQ             } from '../../../modules/ebi-metagenomics/mapseq/main'
+include { MAPSEQ2BIOM        } from '../../../modules/ebi-metagenomics/mapseq2biom/main'
+include { KRONA_KTIMPORTTEXT } from '../../../modules/ebi-metagenomics/krona/ktimporttext/main'
+
+workflow MAPSEQ_OTU_KRONA {
+
+    take:
+    ch_fasta    // channel: [ val(meta), [ bam ] ]
+    ch_dbs      // channel: [ path(fasta), path(tax), path(otu), path(mscluster), val(label) ]
+
+    main:
+
+    ch_versions = Channel.empty()
+
+    db_fasta = ch_dbs[0]
+    db_tax = ch_dbs[1]
+    db_otu = ch_dbs[2]
+    db_mscluster = ch_dbs[3]
+    db_label = ch_dbs[4]
```

Comment:
I really like approach like
```
ch_dbs.multiMap { fasta, tax, otu, mcluster, label ->
        db_fasta: fasta
        db_tax: tax 
        db_otu: otu 
        db_mscluster: mscluster 
        db_label: label
    }.set {
        input
    }
```
and then use it like `MAPSEQ(ch_fasta, [input.db_fasta, input.db_tax, input.db_mscluster] )`
That multiMap is very convenient if you need to assign `meta` to each element in the input and create separate tuples. In your case is not relevant much, but usage of 0,1,2,3,4 might be confusing.

Actually, in that subworkflow makes sense to combine only:
```
ch_dbs.multiMap { fasta, tax, otu, mcluster, label ->
        mapseq_input: [fasta, tax, mscluster]
        mapseq_to_biom_input: [ otu, label ] 
    }.set {
        input
    }
```
and use `MAPSEQ(ch_fasta, input.mapseq_input)` and `MAPSEQTOBIOM(MAPSEQ.out.mseq, input.mapseq_to_biom_input)`

Reply:
this is :hackerman: stuff!! I love it, making this change